### PR TITLE
[pentest/doc] Add building instructions

### PR DESCRIPTION
**Pull request by nasahlpa**
In this commit, instructions how to build the pen. testing framework for the FPGA or the chip are added.

### CODE REVIEW

File: sw/device/tests/penetrationtests/README.md [hunk 1]
```diff
@@ -2,3 +2,28 @@
 
 The penetration testing framework is based on the [Cryptotest](../crypto/cryptotest/README.md) framework.
 The purpose of this framework is to perform side-channel analysis (SCA) and fault injection (FI) attacks on the FPGA as well as on the chip.
+
+To build the penetration testing framework for the ChipWhisperer FPGA board, run:
```
Comment (andrea-caforio) [inline sw/device/tests/penetrationtests/README.md:6]: I assume these instructions are valid for both CW-305 and CW-310?

Comment (nasahlpa) [reply sw/device/tests/penetrationtests/README.md:6]: Good point, no this instruction is just for the CW310 - I'll update the commit. Support for the CW305 is planned at a later stage.

**Review by andrea-caforio — APPROVED** [review]

File: sw/device/tests/penetrationtests/README.md [hunk 2]
```diff
@@ -2,3 +2,28 @@
...
 The purpose of this framework is to perform side-channel analysis (SCA) and fault injection (FI) attacks on the FPGA as well as on the chip.
+
+To build the penetration testing framework for the ChipWhisperer CW310 FPGA board, run:
+```console
+cd $REPO_TOP
+./bazelisk.sh build //sw/device/tests/penetrationtests/firmware:firmware_fpga_cw310_test_rom
+```
+
+Due to code size memory limitations, the firmware for the chip is split into a
+SCA and a FI binary. To build these binaries, run:
```
Comment (vogelpi) [inline sw/device/tests/penetrationtests/README.md:13]: Question do we want to do 1 line per sentence or wrap at 100 chars in this file? Before this PR it was the former, with this PR it's a bit of both.

I suggest checking the Markdown styleguide re some guidance and then adhere to one style only within this file.

**Review by vogelpi — COMMENTED** [review]
Thanks @nasahlpa , can you please align the style with our MarkDown styleguide regarding line wrapping?

**Review by vogelpi — APPROVED** [review]
LGTM, thanks @nasahlpa !

File: sw/device/tests/penetrationtests/README.md (hunk 2 above)
Comment (nasahlpa) [reply sw/device/tests/penetrationtests/README.md]: Oh sorry about that, it seems that we are not following the markdown style guide in ot-sca at all, hence the confusion on my side.
Fixed.

**Comment by nasahlpa** [comment]
Merging - FPGA CI failures are unrelated as this PR only touches a doc.

**Pull request merged**